code/game/objects/items/stacks/wrap.dm

```diff
@@ -18,6 +18,14 @@
 	merge_type = /obj/item/stack/wrapping_paper
 	singular_name = "wrapping paper"
 
+/obj/item/stack/wrapping_paper/interact_with_atom(obj/interacting_with, mob/living/user, list/modifiers)
+	if (istype(interacting_with, /mob/living))
```

Comment:
if you're wondering why this is here, it's because wrapping paper is NOBLUDGEON, so doesn't run most of the attack chain.

Reply:
I imagine you can safely remove `NOBLUDGEON` and it'd still function otherwise

Reply:
I'm going to remove it but if it breaks stuff that's on you!

Reply:
seems ok.